dttools/src/jx_table.c

```diff
@@ -109,6 +109,10 @@ void jx_table_print( struct jx_table *t, struct jx *j, FILE * f, int columns_max
 			line = malloc(10);
 			string_metric(jx_lookup_integer(j,t->name), -1, line);
 			strcat(line, "B");
+		}
+		else if(t->mode == JX_TABLE_MODE_MEGABYTES) {
+			line = malloc(10);
+			string_metric(jx_lookup_integer(j,t->name), -1, line);
```

Comment:
Oh, I see the problem.
This case isn't doing anything different than the `JX_TABLE_MODE_METRIC` case.
Take a look at how string_metric works:
https://github.com/cooperative-computing-lab/cctools/blob/master/dttools/src/stringtools.c#L238

If -1, it chooses the metric prefix automatically.
If >=0 then you can choose the power of 10.

So just set this to always show megabytes.